lib/support/transform/catalog/updateLauncherCatalog.ts

```diff
@@ -70,6 +71,10 @@ export function updateLauncherCatalog(context: HandlerContext,
   };
 }
 
+function getAppropriateNamePrefixForSpringBootVersion(sbVersion: string): string {
+  return sbVersion.startsWith("1.5") ? "previous" : "current";
```

Comment:
We will need to remember to change that when `2.1` becomes the previous version… We could maybe make this a parameter of the update command?

Reply:
Yeah, we should do that more generally when 2.2 comes around. There are a few places this needs to change, but I didn't want to add a general solution until we know how all the versions are going to be handled by the team in general